samples/subsys/portability/cmsis_rtos_v1/philosophers/src/main.c

```diff
@@ -100,19 +100,22 @@ static void print_phil_state(int id, const char *fmt, int32_t delay)
 	int prio = osThreadGetPriority(osThreadGetId());
 
 	set_phil_state_pos(id);
+	char state[80];
+	int p = 0;
 
-	printk("Philosopher %d [%s:%s%d] ",
-	       id, prio < 0 ? "C" : "P",
-	       prio < 0 ? "" : " ",
-	       prio);
+	p += snprintk(state + p, 80 - p, "Philosopher %d [%s:%s%d] ",
+		     id, prio < 0 ? "C" : "P",
+		     prio < 0 ? "" : " ",
+		     prio);
 
 	if (delay) {
-		printk(fmt, delay < 1000 ? " " : "", delay);
+		p += snprintk(state+p, 80 - p, fmt, delay < 1000 ? " " : "", delay);
```

Comment:
Nit: For the sake of consistency

```suggestion
		p += snprintk(state + p, 80 - p, fmt, delay < 1000 ? " " : "", delay);
```

Reply:
oops. whitespace is hard.